api/src/main/java/org/openmrs/OrderGroup.java

```diff
@@ -208,4 +210,26 @@ public void setId(Integer id) {
 		setOrderGroupId(id);
 	}
 	
+	/**
+	 * Gets the ParentOrderGroup to other OrderGroups, to maintain linkages between groups and
+	 * support group nesting
+	 * 
+	 * @param returns the ParentOrderGroup
+	 * @since 2.3.0..
+	 */
+	public OrderGroup getParentOrderGroup() {
+		return parentOrderGroup;
+	}
+	
+	/**
+	 * Sets the ParentOrderGroup to other OrderGroups, to maintain linkages between groups and
+	 * support group nesting
+	 * 
+	 * @param ParentOrderGroup The ParentOrderGroup to set
```

Comment:
Please follow this convention below:
@param parentOrderGroup the parent order group to set